values.yaml

```
@@ -111,6 +112,14 @@ global:
     # clients and servers and only accept HTTPS connections.
     httpsOnly: true
 
+  # [Enterprise Only] consulNamespacesEnabled indicates that you are running
+  # Consul v1.7+ with a valid Consul Enterprise license and would like to make
```

```suggestion
  # Consul Enterprise v1.7+ with a valid Consul Enterprise license and would like to make
```

You have to run the consul enterprise binary so make that clear

---

Updated.